src/Results.js

```diff
@@ -3,10 +3,10 @@ import { Link } from 'react-router-dom';
 
 import './Results.css'
 
-const Results = ( {favoriteCats, favoriteDogs, animalChoice }) => {
+const Results = ( {favoriteCats, favoriteDogs, animalChoice, clearFavorites }) => {
   const [winner, setWinner] = useState('');
   const [images, setImages] = useState([]);
-
+  
   const findWinner = () => {
 
     if(favoriteCats.length > favoriteDogs.length ) {
```

Comment:
I love how simple this determination is! Good work not over complicating this conditional.